scalding-jdbc-macros/src/main/scala/com/twitter/scalding/jdbc/macros/JDBCMacro.scala

```diff
@@ -0,0 +1,25 @@
+package com.twitter.scalding.jdbc.macros
+
+import scala.language.experimental.macros
+import com.twitter.scalding.jdbc.macros.impl._
+import com.twitter.scalding.jdbc.{ ColumnDefinitionProvider, JDBCTypeInfo }
+
+sealed trait ScaldingJdbcAnnotation
+// This is the size in characters for a string field
+// For integers its really for display purposes
+@scala.annotation.meta.getter
+class size(val size: Int) extends annotation.StaticAnnotation with ScaldingJdbcAnnotation
+
+@scala.annotation.meta.getter
```

Comment:
add comments on each of these, along with a top one above as to what they are for